tasks-by-users/Tauqeer-Ahmed-99/01-Responsive Resume Website/index.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Responsive CV using HTML CSS</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"
+      integrity="sha512-5A8nwdMOWrSz20fDsjczgUidUBR8liPYU+WymTZP1lmY9G6Oc7HlZv156XqnsgNUzTyMefFTcsFH/tnJE/+xBg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <div class="container">
+      <div class="left-side">
+        <div class="profiletext">
+          <div class="imgbox">
+            <img src="images/tauqeer.jpg" alt="Profile image" />
+          </div>
+          <h2>Tauqeer Khan<br /><span>Web Developer</span></h2>
+        </div>
+        <div class="contact">
+          <h3 class="title">Contact Info</h3>
+          <ul>
+            <li>
+              <span class="icon"
+                ><i class="fa fa-mobile" aria-hidden="true"></i
+              ></span>
+              <span class="text">+91 8879998633</span>
+            </li>
+            <li>
+              <span class="icon"
+                ><i class="fa fa-envelope-o" aria-hidden="true"></i
+              ></span>
+              <span class="text">tauqueerrkhan@gmail.com</span>
+            </li>
+            <li>
+              <span class="icon"
+                ><i class="fa fa-globe" aria-hidden="true"></i
```

Comment:
Looks like a prettier problem
cc/ @Naufil-Shemle , @coder-mezab , @fayez-baig

Reply:
yes to me too..